src/pestuary.py

```diff
@@ -1,3 +1,7 @@
+import inspect
+from functools import wraps
+import click
+import jsonpickle
```

Comment:
I'm not sure which of these are external, but we need to include the ones that are on `requirements.txt`

Reply:
sorry this isn't in the PR - it says outdated and the code in the PR doesn't have this in it